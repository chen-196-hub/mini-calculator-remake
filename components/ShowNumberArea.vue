<template>
  <v-card class="text-center py-3 my-3">
    {{ props.number || 0 }}
  </v-card>
</template>

<script setup lang="ts">
interface Props {
  number: string
}
const props = defineProps<Props>()
</script>
