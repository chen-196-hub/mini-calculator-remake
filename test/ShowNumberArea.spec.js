import Vuetify from 'vuetify'
// Utilities
import { createLocalVue, mount } from '@vue/test-utils'
import ShowNumberArea from '@/components/ShowNumberArea.vue'

const localVue = createLocalVue()

describe('CustomNavigationDrawer.vue', () => {
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  it('should have a custom title and match snapshot', () => {
    const wrapper = mount(ShowNumberArea, {
      localVue,
      vuetify,
    })

    expect(wrapper.text()).toBe('0')
  })
})
