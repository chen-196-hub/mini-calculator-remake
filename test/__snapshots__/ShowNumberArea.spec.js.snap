// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowNumberArea.vue should have a custom number and match snapshot 1`] = `
"<div class="text-center py-3 my-3 v-card v-sheet theme--light" number="100">
  <div class="v-card__title"> </div>
</div>"
`;
